modules/git/commit.go

```diff
@@ -8,6 +8,7 @@ package git
 import (
 	"bufio"
 	"bytes"
+	"code.gitea.io/gitea/models"
```

Comment:
import order

Reply:
add this line betwen current L23 and L24 (leave space betwen "strings" and "gopkg.in/...")